inventory/group_vars/mergers

```diff
@@ -18,6 +18,3 @@ bonnyci_zuul_merger_apache_vhosts:
       ScriptAlias /p/ /usr/lib/git-core/git-http-backend/
 
 bonnyci_zuul_merger_git_dir: /var/lib/zuul/git
-
-zuul_components:
```

Comment:
I don't think it's a problem here - but conceptually lists don't merge when they are overridden. If you were to put a merger on the same host as something else the zuul_components var would be overridden.